tfhe/src/integer/server_key/radix/tests.rs

```diff
@@ -311,14 +336,18 @@ fn integer_unchecked_bitand(param: ClassicPBSParameters) {
 
         let clear_1 = rng.gen::<u64>() % modulus;
 
+        let start = time::Instant::now();
```

Comment:
Do we want to keep these timings prints ?

Reply:
No you're right. It must be a leftover of my debug phase.